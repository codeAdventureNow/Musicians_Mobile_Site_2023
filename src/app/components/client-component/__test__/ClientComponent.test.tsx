import { render } from '@testing-library/react';
import ClientComponent from '../client-component';

describe('Client Component', () => {
  it('should render the "Client Componenent"', () => {
    const { asFragment } = render(<ClientComponent />);

    expect(asFragment()).toMatchSnapshot();
  });
});
