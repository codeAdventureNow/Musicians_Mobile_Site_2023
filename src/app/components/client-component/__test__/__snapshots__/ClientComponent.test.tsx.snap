// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client Component should render the "Client Componenent" 1`] = `
<DocumentFragment>
  <div>
    <ul
      class="flex"
    />
  </div>
</DocumentFragment>
`;
